lib/elements/prompt.js

```diff
@@ -16,7 +16,7 @@ class Prompt extends EventEmitter {
     this.out = process.stdout;
 
     const rl = readline.createInterface(this.in);
-    readline.emitKeypressEvents(this.in, this.rl);
+    readline.emitKeypressEvents(this.in, rl);
```

Comment:
I still have to look into this. The second param to `emitKeypressEvents` is optional - do we even need it?

Also have to dig deeper into `createInterface` options like `crlfDelay` and `terminal`
https://nodejs.org/api/readline.html#readline_readline_createinterface_options